swf/src/types/bevel_filter.rs

```diff
@@ -33,6 +33,11 @@ impl BevelFilter {
     pub fn num_passes(&self) -> u8 {
         (self.flags & BevelFilterFlags::PASSES).bits()
     }
+
+    pub fn scale(&mut self, x: f32, y: f32) {
```

Comment:
I think the more Rust-y way is to avoid mutation of structs, but instead consume and return a new struct. Something like:
```rs
pub fn scale(mut self, x: f32, y: f32) -> Self {
    self.blur_x *= Fixed16::from_f32(x);
    self.blur_y *= Fixed16::from_f32(y);
    self
}
```

Reply:
I'm not sure. The enum method would look like this:

```rust
        match self {
            Filter::BevelFilter(filter) => filter.scale(x, y).into(),
            Filter::BlurFilter(filter) => filter.scale(x, y).into(),
            Filter::DropShadowFilter(filter) => filter.scale(x, y).into(),
            Filter::GlowFilter(filter) => filter.scale(x, y).into(),
            Filter::GradientBevelFilter(filter) => filter.scale(x, y).into(),
            Filter::GradientGlowFilter(filter) => filter.scale(x, y).into(),
            _ => self,
        }
```

It doesn't feel rusty to me

Reply:
Hmm, yeah. I guess avoiding those `.into()` is indeed preferred.